### ISSUE TITLE
Formatting issue in the documentation

### ISSUE THREAD

**Issue by bheesham** [jazzband/django-simple-history#1010]
**Describe the bug**

The formatting in this section is incorrect: https://django-simple-history.readthedocs.io/en/latest/common_issues.html#using-custom-onetoonefields

```
If you are using a custom OneToOneField that has additional arguments and receiving
the the following ``TypeError``::

.. code=block:: python

    TypeError: __init__() got an unexpected keyword argument

```

It should be:

```
If you are using a custom OneToOneField that has additional arguments and receiving
the the following ``TypeError``::

    TypeError: __init__() got an unexpected keyword argument

```

**To Reproduce**
1. View docs here: https://django-simple-history.readthedocs.io/en/latest/common_issues.html#using-custom-onetoonefields
2. Note how the documentation includes the RST text.

**Expected behavior**
The documentation should not have included `.. code=block:: python` in the generated output.

**Screenshots**
![issue django-simple-history generated docs](https://user-images.githubusercontent.com/171007/177685105-5f1d44af-da15-4513-a3c6-a47786ccdd49.png)

**Environment (please complete the following information):**
 - OS: Fedora
 - Browser (if applicable): Chrome
 - Django Simple History Version: 3.1.1
 - Django Version: 4.0.6
 - Database Version: SQLite 3

**Additional context**
None.

**Comment by bheesham** [comment]
I'm currently prepping a PR.

**Comment by joshuadavidthomas** [comment]
Closed in #1011
